desc/objectives/_coils.py

```diff
@@ -607,6 +607,128 @@ def compute(self, params, constants=None):
         return out
 
 
+class CoilCurrentLength(CoilLength):
```

Comment:
I don't think it makes sense to inherit from `CoilLength`, I think we should inherit from `_CoilObjective` instead. This doesn't save much code and it feels weird to overwrite the normalization already set by `CoilLength`. And thematically, I feel `CoilLength` and `CoilCurrentLength` should be the same "generation" in the family tree.

Reply:
Never mind